contrib/ansible/cri-containerd.yaml

```diff
@@ -16,6 +16,11 @@
     - name: "Start CRI-Containerd"
       systemd: name=cri-containerd daemon_reload=yes state=started enabled=yes
 
+    - name: "Load br_netfilter kernel module"
```

Comment:
this should be in bootstrap_ubuntu I feel.

Reply:
Why don't we need this for centos? We need this kernel module on any linux distro, right?
Or else there won't even be `net.bridge` sysctl.

Reply:
my bad. It works. 
bridge                107151  2 br_netfilter,ebtable_broute